### DISCUSSION TITLE
Unable to Flash Inav 7 or 8 from 6

### DISCUSSION THREAD

**Discussion by Rojohnso13** [iNavFlight/inav#10564] [General]
Hope somebody can help. I seem to be unable to flash my new fc (Speedybee f405wing mini) to any other versions of Inav. It came pre configured with 6.1.1. I have tried both 7 and 8 with the same result. I have tried all manner of booting sequences in Dfu but cant get it to work. The Auto select target box is greyed out and the dropdown lists for firmware and board choices shows as offline. The board loads fine in Configurator 6 and everything works but I have the same issue when on the flashing page. No dropdowns and offline.

**Comment by MrD-RC** [comment]
If it shows offline. It sounds like there is an internet access problem. Either your computer can't see the internet or a firewall is blocking Configurator.

**Reply by Rojohnso13** [reply]
I did think that but after turning off my internet security (Comodo) and windows firewall the issue still persists

**Reply by Rojohnso13** [reply]
I ended up flashing the Fc manually in the end. Thank you for your help

**Reply by terryfritz99** [reply]
I had the same going to INAV8 RC3 and I suggest trying ImpulseRC Driver Fixer. My laptop didn't want to see the FC in DFU mode when I used Config release 8.
